packages/flutter/lib/src/widgets/actions.dart

```diff
@@ -650,42 +648,22 @@ class _ActionsState extends State<Actions> {
   }
 
   void _handleActionChanged(Action<Intent> action) {
-    assert(enabledState.containsKey(action));
-    final bool actionEnabled = action.enabled;
-    if (enabledState[action] == null || enabledState[action] != actionEnabled) {
-      setState(() {
-        enabledState[action] = actionEnabled;
-        // Generate a new key so that the marker notifies dependents.
-        rebuildKey = Object();
-      });
-    }
+    // Generate a new key so that the marker notifies dependents.
+    rebuildKey = Object();
```

Comment:
Wrap this in setState to ensure it rebuilds?

Reply:
Thanks, done.